index.rst

```diff
@@ -0,0 +1,36 @@
+.. CVE Binary Tool documentation master file, created by
```

Comment:
We already have an index.rst in doc/ and should not have a second one in the main directory, so this is not needed.